libnopegl/src/nopegl.h.in

```diff
@@ -718,7 +718,7 @@ NGL_API struct ngl_ctx *ngl_create(void);
  *
  * @return 0 on success, NGL_ERROR_* (< 0) on error
  */
-NGL_API int ngl_configure(struct ngl_ctx *s, struct ngl_config *config);
+NGL_API int ngl_configure(struct ngl_ctx *s, const struct ngl_config *config);
```

Comment:
can you make it const in the `c-api.md` documentation as well?

Reply:
Done.